manifest.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "Video Speed Controller",
+  "name": "Video Speed Controller Redesign",
```

Comment:
The name doesn't have to change and the version can be changed to 6.4 not 2.0

Reply:
Ah, I apologize I didn't intend to commit that change to this pull request. I was in the process of replacing this repository with an update based on @igrigorik 's last reply when you commented. Please see the new Design Update pull request here: #705